jekyll/_includes/site/header.html

```diff
@@ -1,2 +1,4 @@
 <h1>PostingPro</h1>
 <p class="tagline">Build inclusive job postings that engage quality candidates!</p>
+<p>Either start a new job posting or build from an existing template. As you type, the text will be analyzed for grade reading level and problamatic phrases that might deter the right job seeker from applying. If the reading level is too high, it will be flagged by a red line.</p>
+<p><span class="bold">Keep the reading level at or below the required education for the position so that it is more inclusive.</span><br>For example, if the job does not require a college degree then the reading level should not be above 12th grade. Remember, the average adult’s reading level is 7th/8th grade. </p>
```

Comment:
Did you see people clicking on it and that they were surprised? I like a link because it's really hard to explain readability, how to improve, and why it's important in a couple sentences.

Anyways, I'd phrase the last part as "the average adult is (most?) comfortable reading at a 7th to 8th grade level", it sounds more positive. And maybe the "reading level should not be above 12th grade" as "reading level should be below 12th grade"?